assets/js/sticky-menu.js

```diff
@@ -9,6 +9,7 @@ define( ['jquery', 'underscore'], function ( $, _ ) {
 
 	var config = {
 		bodyStickyClass:       'js-sticky-navigation', // Present, when sticky is enabled in customizer.
+		bodyStickyLayoutClass: 'js-sticky-all', // Class used for triggering the sticky menu layout.
```

Comment:
@krumpak We discussed the naming of this variable yesterday... I thought about it and I think that "Layout" is not good in this situation, so maybe "Visibility" would be a better word?